src/services/platform.service.js

```diff
@@ -44,10 +44,21 @@ export const PACKAGE_MANAGER_CMD = path.join(
 // Forward the host env, and append the
 // project's .bin directory to PATH to allow
 // package scripts to function properly.
-export const getBaseProjectEnvironment = (projectPath: string) => ({
-  ...window.process.env,
+/**
+ * For running tasks in a cross-platform manner, this helper does a few things:
+ *  - Forward the host environment
+ *  - Append the project's .bin directory to PATH to allow package scripts to
+ *    function properly.
+ *  - Add `FORCE_COLOR: true`, so that terminal output includes color codes.
+ */
+export const getBaseProjectEnvironment = (
+  projectPath: string,
+  currentEnvironment: Object = window.process.env
+) => ({
+  ...currentEnvironment,
+  FORCE_COLOR: true,
```

Comment:
It feels a bit weird to have this here, since it isn't platform-specific... but I like the guarantee of knowing it will always be set (since we'll always need to invoke this method to get platform-specific environment info).

Reply:
it indeed does feel weird to set it here: if at one point we want to run a command (say run a generator for a new react component) and want to parse the output, there will be the control characters in the middle

Reply:
@mathieudutour that feels like somewhat of an edge case, no? If not, it's at least a ways down the development roadmap, since it first requires integrating custom tasks. I could foresee a toggle when adding a custom task to disable color output, perhaps with a note about why you may want to do that (like the example you gave). Maybe worth adding a `NOTE`/`TODO` here to that effect, but I think for now this is the most logical place for this flag.

Reply:
I don't mind too much but I usually prefer to have to "enabling" things explicitly instead of "disabling" them.

Reply:
Yeah, I do see that point. it feels a bit weird to me too. But I think the issue is that it can be easy to forget to enable something, and for it to go unnoticed in code-review, leading to uncolored terminals. I think this is one of those cases where the "tidy" thing is also the one most likely to lead to poorer user experiences.

I think if/when we need to strip control characters out, we can create a new "lower-level" function that can be used without control characters.